crates/refunder/src/arguments.rs

```diff
@@ -45,6 +54,7 @@ impl std::fmt::Display for Arguments {
         writeln!(f, "min_slippage_bps: {}", self.min_slippage_bps)?;
         writeln!(f, "db_url: SECRET")?;
         writeln!(f, "node_url: {}", self.node_url)?;
+        writeln!(f, "ethflow_contract: {}", self.ethflow_contract)?;
```

Comment:
```suggestion
        writeln!(f, "ethflow_contract: {:?}", self.ethflow_contract)?;
```
With `Display` formatting: `ethflow contract: 0x26c3…2830`
With `Debug` formatting: `ethflow contract: 0x26c3801b4840dab317cedfd7aeaf9f45fdd22830`
I suggest the same change for the `autopilot` crate as well.